doc/meetings/2016-07-13.md

```diff
@@ -0,0 +1,117 @@
+# Attendees
+
+* Myles Borins
+* Michael Dawson
+* Jeremiah Senkpiel
+* Ali Sheikh
+* Michael Hablich
+* Franziska Hinkelmann
+* Rod Vagg
+
+# Agenda
+
+* Managing v8 floating patches
+* v4.5
+* Exporting ssl symbols on windows
+
+# Discussion
+
+## Managing v8 floating patches - https://github.com/nodejs/LTS/issues/111
+
+* V8 team willing to have place where floating patching can land
+* Overlap of 2-3 weeks were still maintained that might cause conflicts
+* would need to create separate branch for node.
+* Why not create fork
+* Notes from Ben N
+  * What timeframe for using google infrastructure
+  * We have our own infra and can run from Node tree
+
+### Infrastructure
+* Chromium infrastructure team is moving fast and deprecating lots of things so may
```

Comment:
Chromium infrastructure is designed for the chromium's evergreen release model. Lots of new features are being added (e.g. swarming being added at the moment) and generally it is not a goal that the build infrastructure will be able to support building and testing V8 older than a year or even a few months.